schema/schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
```

Comment:
You should probably set `"additionalProperties": false,` on root level as that gives stricter validation.